dd-sdk-android/src/main/kotlin/com/datadog/android/v2/core/internal/net/DataOkHttpUploader.kt

```diff
@@ -56,6 +56,12 @@ internal class DataOkHttpUploader(
                 "Unable to upload batch data.",
                 e
             )
+            internalLogger.log(
+                InternalLogger.Level.DEBUG,
+                InternalLogger.Target.USER,
+                "Unable to upload batch data.",
+                e
+            )
```

Comment:
you can combine the targets and make just one call: 
```
            internalLogger.log(
                InternalLogger.Level.ERROR,
                listOf(InternalLogger.Target.MAINTAINER, InternalLogger.Target.USER)
                "Unable to upload batch data.",
                e
            )
```

Reply:
The only reason we might not want to do that is because of the difference in level -- this isn't really an error from the user's perspective, as we're just going to retry. But we need the extra info in the log if the user is experiencing significant trouble uploading data.

Alternately we can set the level to Debug across the board for this error? Thoughts @0xnm ?

Reply:
imo let's keep it different: debug - for the user, error for us. Having error at least is more visible/noticeable in logcat.